doc/rados/configuration/osd-config-ref.rst

```diff
@@ -384,22 +384,16 @@ Operations
 :Description: This sets the type of queue to be used for prioritizing ops
               in the OSDs. Both queues feature a strict sub-queue which is
               dequeued before the normal queue. The normal queue is different
-              between implementations. The original PrioritizedQueue (``prio``) uses a
-              token bucket system which when there are sufficient tokens will
-              dequeue high priority queues first. If there are not enough
-              tokens available, queues are dequeued low priority to high priority.
-              The WeightedPriorityQueue (``wpq``) dequeues all priorities in
-              relation to their priorities to prevent starvation of any queue.
-              WPQ should help in cases where a few OSDs are more overloaded
-              than others. The new mClock based OpClassQueue
-              (``mclock_opclass``) prioritizes operations based on which class
+              between implementations. The WeightedPriorityQueue (``wpq``)
+              dequeues all priorities in relation to their priorities to prevent
```

Comment:
"dequeues all priorities in relation to their priorities" seems confusing. I wonder if it should read "dequeues operations in relation to their priorities".